extensions/amp-connatix-ps-player/OWNERS

```diff
@@ -0,0 +1,10 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [{name: 'rarespreda'}, {name: 'ampproject/wg-ui-and-a11y'}],
```

Comment:
ampproject/wg-ui-and-a11y ?

Reply:
same as it is in extensions/amp-connatix-player/OWNERS